src/inspector_socket_server.cc

```diff
@@ -25,9 +25,18 @@ void Escape(std::string* string) {
 }
 
 std::string GetWsUrl(const std::string& host, int port, const std::string& id) {
-  char buf[1024];
-  snprintf(buf, sizeof(buf), "%s:%d/%s", host.c_str(), port, id.c_str());
-  return buf;
+  // Host is valid (socket was bound) so colon means it's a v6 IP address
```

Comment:
I was not aware of that code. I will share the code for generating the URL. It will still be slightly different - e.g. in json/list response it echoes IP address the client used to access the server while other places will print the host name specified from the command line (`--inspect=0.0.0.0` will respond with 127.0.0.1 or external IP depending on the client but print out 0.0.0.0).